gnd/src/main/java/com/google/android/gnd/model/observation/TextResponse.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.model.observation;
+
+import com.google.android.gnd.model.form.Field;
+import java8.util.Optional;
+
+/** A user provided response to a text {@link Field}. */
+public class TextResponse implements Response {
+
+  private String text;
+
+  public TextResponse(String text) {
+    this.text = text;
+  }
+
+  public String getText() {
+    return text;
+  }
+
+  @Override
+  public String getSummaryText(Field field) {
```

Comment:
No for text I think it's ok if they're the same. This was added so that multiple choice can return codes instead of labels. We can probably even remove these once we show the record instead of the abbreviated result cards.